Build/CMakeLists.txt

```diff
@@ -14,6 +14,9 @@ option(CROSS_PLATFORM_DETERMINISTIC "Cross platform deterministic" OFF)
 # When turning this option on, the library will be compiled for ARM (aarch64-linux-gnu), requires compiling with clang
 option(CROSS_COMPILE_ARM "Cross compile to aarch64-linux-gnu" OFF)
 
+# When turning this option on, Jolt will be compiled as a shared library and public symbols will be exported.
+option(COMPILE_AS_SHARED_LIBRARY "Compile Jolt as a shared library" OFF)
```

Comment:
The 'Distribution' build currently gives a compile error if you turn this on (because I'm doing some tricks with DebugRenderer.h).

Reply:
Yeah, I managed to make those tricks work, but it's not pretty.